Simplenote/Classes/SPSidebarContainerViewController.m

```diff
@@ -340,12 +349,12 @@ - (void)panGestureWasRecognized:(UIPanGestureRecognizer *)gesture
         BOOL didBecomeVisible = self.isSidebarVisible;
 
         [self.animator addCompletion:^(UIViewAnimatingPosition finalPosition) {
+            weakSelf.isPanningActive = NO;
             [weakSelf endSidebarTransition:didBecomeVisible];
```

Comment:
Because this is still obj-c I would probably use `__strong __typeof(weakSelf) strongSelf = weakSelf;` to make sure it will never become _nil_ when `endSidebarTransition:` is called. What do you think?

Reply:
Updating, **thank you**!!!!